### PR TITLE
Update the TODOs in the README.

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

File: firestore/testapp/README.md [hunk 1]
```diff
@@ -13,7 +13,19 @@ The testapp performs the following:
 -   Gets a pointer to firebase::Auth, and signs in anonymously. This allows the
     testapp to access a Firebase Firestore instance with authentication rules
     enabled.
--   TODO(varconst): describe the Firestore-specific logic
+-   Initializes a Firestore instance, and sets its logging level to
```
Comment (var-const) [inline firestore/testapp/README.md:16]: Ultranit: I think the comma before `and` is extraneous.

File: firestore/testapp/README.md [hunk 2]
```diff
@@ -13,7 +13,19 @@ The testapp performs the following:
...
+    `kLogLevelDebug` in order to see debug messages in the logs.
+-   Tests that it can create Timestamp, SnapshotMetadata, and GeoPoint objects.
```
Comment (var-const) [inline firestore/testapp/README.md:18]: Nit: can you wrap `Timestamp`, etc. in backticks for consistency.

File: firestore/testapp/README.md [hunk 3]
```diff
@@ -13,7 +13,19 @@ The testapp performs the following:
...
+-   Creates a collection, and a document inside that collection.
```
Comment (var-const) [inline firestore/testapp/README.md:19]: Same ultranit re. the comma.

**Review by var-const — APPROVED** [review]

File: firestore/testapp/README.md (hunk 1 above)
Comment (ehsannas) [reply firestore/testapp/README.md]: Done

File: firestore/testapp/README.md (hunk 2 above)
Comment (ehsannas) [reply firestore/testapp/README.md]: Done.

File: firestore/testapp/README.md (hunk 3 above)
Comment (ehsannas) [reply firestore/testapp/README.md]: Done

**Pull request merged**